<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_start">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe 宋体 Std L</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_restart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe 宋体 Std L</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>RESTART</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_scale">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe 宋体 Std L</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>SCALE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_end">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe 宋体 Std L</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>END</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>48</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Adobe 宋体 Std L</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>camera 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>camera 2</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_select">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>48</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Adobe 宋体 Std L</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>REHABILITATION</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_image">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="actionDeadlift">
   <property name="text">
    <string>Deadlift</string>
   </property>
  </action>
  <action name="actionLat_Pulldown">
   <property name="text">
    <string>Lat Pulldown</string>
   </property>
  </action>
  <action name="actionSquats">
   <property name="text">
    <string>Squats</string>
   </property>
  </action>
  <action name="actionBench_Press">
   <property name="text">
    <string>Bench Press</string>
   </property>
  </action>
  <action name="actionDumbbel_Shoulder_Press">
   <property name="text">
    <string>Dumbbel Shoulder Press</string>
   </property>
  </action>
  <action name="actionDumbbell_Lateral_Raises">
   <property name="text">
    <string>Dumbbell Lateral Raises</string>
   </property>
  </action>
  <action name="actionShoulder_rehabilitation">
   <property name="text">
    <string>Shoulder rehabilitation</string>
   </property>
  </action>
  <action name="actionKnee_rehabilitation">
   <property name="text">
    <string>Knee rehabilitation</string>
   </property>
  </action>
  <action name="actionLumbar_spine_rehabilitation">
   <property name="text">
    <string>Lumbar spine rehabilitation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
